### ISSUE TITLE
Comandos !tps e !atps não estão funcionando no momento

### ISSUE THREAD

**Issue by danecool** [victorsouzaleal/lbot-whatsapp#43]
Boa tarde Victor,

Primeiramente parabéns pelo BOT, foi o mais completo que já encontrei. De fato, parabéns mesmo, e claro, muito obrigado pela oportunidade.

Fiz diversos testes e tentei encontrar o problema de conversão de texto em sticker, porém não encontrei.

![image](https://github.com/user-attachments/assets/44ad5872-5113-49c3-a132-eb9868a0cac2)

Motivo: Houve um erro no servidor para transformar texto em imagem/gif.

![image](https://github.com/user-attachments/assets/3c22dd82-7988-456b-a8e0-9105f7d344f1)

Abraços e tudo de bom!

**Comment by victorsouzaleal** [comment]
A api que eu usava saiu do ar e não tenho outra pra substituir no momento

**Comment by danecool** [comment]
Tem aquela attp

**Comment by danecool** [comment]
É do spyder, aparentemente simples de aplicar....
api https://api.spiderx.com.br/ ( free )
js https://github.com/guiireal/takeshi-bot/blob/main/src/commands/member/attp.js

call:
exports.SPIDER_API_BASE_URL = "https://api.spiderx.com.br/api";
exports.SPIDER_API_TOKEN = "seu_token_aqui";
exports.ONLY_GROUP_ID = "";

**Comment by danecool** [comment]
Sim, igual a de ACRCLOUD e DEEPGRAM... cada usuário cria a conta e atualiza os dados no arquivo .env, depois no caso.
Eu uso com outro bot ( https://github.com/guiireal/takeshi-bot ) e está normal.

**Comment by victorsouzaleal** [comment]
> Sim precisa de token, igual a de ACRCLOUD e DEEPGRAM... cada usuário cria a conta e atualiza os dados no arquivo .env, depois no caso. Eu uso com outro bot ( https://github.com/guiireal/takeshi-bot ) e está normal.

Eu tento ter o mínimo possível de comando que precisa de token, vou ter que achar um que não precise.
